dockerfiles/base/scripts/base/commands/cmd_config.sh

```diff
@@ -116,6 +116,7 @@ generate_configuration_with_puppet() {
     # Handle override/addon
     if [ -d "/repo/dockerfiles/init/addon" ]; then
       WRITE_PARAMETERS+=" -v \"${CHE_HOST_DEVELOPMENT_REPO}/dockerfiles/init/addon/addon.pp\":/etc/puppet/manifests/addon.pp:ro"
+      WRITE_PARAMETERS+=" -v \"${CHE_HOST_DEVELOPMENT_REPO}/dockerfiles/init/addon/modules/\":/etc/puppet/addon/:ro"
```

Comment:
1) will add check.
2) no because `/etc/puppet/modules` is mounted in dev mode already so another mount in that location will remove previous mount content. I actually did as you saying first time but it failed then I realized that i need to add it separately and include in `modulepath`